p2p_wallet/Scenes/Main/InvestSolend/InvestView.swift

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 P2P Validator Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+import SwiftUI
+
+struct InvestView: View {
+    var body: some View {
+        NavigationView {
+            InvestSolendView(viewModel: try! .init(mocked: false))
+                .navigationBarHidden(true)
+        }
+    }
+}
```

Comment:
why do you need this view? You embed InvestSolendView inside navigationView inside itself

Reply:
@TrGiLong wrote this. Can you explan?